modules/multicast/receiver.c

```diff
@@ -803,6 +803,17 @@ int mcreceiver_alloc(struct sa *addr, uint8_t prio)
 	if (!addr || !prio)
 		return EINVAL;
 
+	if (!IN_MULTICAST(sa_in(addr))
+#ifdef HAVE_INET6
+	    && !IN6_IS_ADDR_MULTICAST(&addr->u.in6.sin6_addr)) {
+#else
+	    ) {
+#endif
+		warning ("multicast receiver: address %J not a multicast"
+			 " address\n", addr);
+		return EINVAL;
+	}
+
```

Comment:
Module multicast should also work for unicast IP addresses.

Reply:
Done.